tests/CMakeLists.txt

```diff
@@ -34,10 +30,6 @@ add_subdirectory(${CMAKE_BINARY_DIR}/googletest-src
 
 # }}}
 
-# Disable errors for warnings so that we can run tests even with some warnings
-string(REGEX REPLACE "-Werror[^ ]*" "" CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
```

Comment:
We probably still want this for the tests. I found out that you can also just to `-Wno-error`. I'd say we could add that option to all `Debug` builds, not just `Coverage`, since it's a good idea for development.

Reply:
Added [here](https://github.com/jaagr/polybar/pull/1510/files#diff-1c7def276df20ca1f92e633223fef5fcR47)